elastic/wazuh_cf_kibana.sh

```diff
@@ -271,15 +271,18 @@ install_plugin(){
   echo "Installing app" >> /tmp/deploy.log
   if [[ ${EnvironmentType} != 'sources' ]] || [[ ${BRANCH} == "" ]]
   then
+    cd /usr/share/kibana
     sudo -u kibana /usr/share/kibana/bin/kibana-plugin install ${plugin_url}
   else
+    cd /usr/share/kibana
     sudo -u kibana /usr/share/kibana/bin/kibana-plugin install file://$BUILD_SRC/$APP_FILE
   fi
+  cd /tmp
   echo "App installed!" >> /tmp/deploy.log
   echo "Redirecting to Wazuh app " >> /tmp/deploy.log
   # Set Wazuh app as the default landing page
-  echo "server.defaultRoute: /app/wazuh" >> /etc/kibana/kibana.yml
-  # Redirect Kibana welcome screen to Discover
+  #echo "server.defaultRoute: /app/wazuh" >> /etc/kibana/kibana.yml
```

Comment:
Please describe the reason why this was removed.

Reply:
Testing typo, fixed in 9fe55910f91e2df6b406f8a17329fd99cfa3f208